dev-client/src/screens/HomeScreen/components/TemporarySiteCallout.tsx

```diff
@@ -41,6 +41,15 @@ type Props = {
 export const TemporarySiteCallout = ({coords, closeCallout}: Props) => {
   const {t} = useTranslation();
   const navigation = useNavigation();
+  const [siteElevation, setSiteElevation] = useState('');
+
+  useMemo(async () => {
+    const elevation = await getElevation(
+      parseFloat(coords.latitude.toFixed(5)),
+      parseFloat(coords.longitude.toFixed(5)),
+    );
+    setSiteElevation(t('site.elevation_value', {value: elevation, units: 'm'}));
```

Comment:
issue: if i'm tracing the code right, i think if the API call fails this will render the string "unknown m" instead of "unknown"?